.github/golangci-lint.config.yaml

```diff
@@ -13,6 +13,7 @@ linters-settings:
       - asmdecl
       - assign
       - atomic
+      - bodyclose
```

Comment:
Doesn't this need to go in the linters section?

The docs suggest that [govet](https://golangci-lint.run/docs/linters/configuration/#govet) doesn't have "bodyclose", but it does have a pure linter for this https://golangci-lint.run/docs/linters/configuration/#bodyclose

Reply:
Yip, my bad. Fixed